code/entities/machine/soda-fountain/interactable/Dispenser.cs

```diff
@@ -0,0 +1,94 @@
+﻿using Sandbox;
+using Sandbox.util;
+using Sandbox.Utility;
+
+namespace Cinema;
+
+public partial class Dispenser : BaseInteractable
+{
+    static private float DispenseTime = 3.0f;
```

Comment:
Nit: Proper constants should be `=>` so it's readonly.

Reply:
Updated DispenseTime to be readonly